src/helpers.cr

```diff
@@ -32,4 +63,64 @@ module Shards::Helpers
       name
     {% end %}
   end
+
+  def self.privilege_enabled?(privilege_name : String) : Bool
+    {% if flag?(:win32) %}
```

Comment:
I'd suggest to put those (heap helpers) in a `Windows` namespace or mark them in other way (prefix, perhaps?) as windows-specific.